compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.cpp

```diff
@@ -476,28 +477,37 @@ static LogicalResult populateCanonicalOffsetsSizesAndStrides(
 
   OpFoldResult zero = builder.getIndexAttr(0);
   OpFoldResult one = builder.getIndexAttr(1);
+  Value cZero = builder.createOrFold<arith::ConstantIndexOp>(loc, 0);
```

Comment:
If we're just creating a Value anyway,
```suggestion
  Value cZero = builder.create<arith::ConstantIndexOp>(loc, 0);
```